Doc/whatsnew/3.7.rst

```diff
@@ -483,138 +782,458 @@ The :func:`~binascii.b2a_uu` function now accepts an optional *backtick*
+
+``executor.submit()`` now raises a :exc:`RuntimeError` if called during
+interpreter shutdown.
+(Contributed by Mark Nemec in :issue:`33097`.)
+
+
+configparser
+------------
+
+The :class:`~configparser.ConfigParser`` constructor now uses ``read_dict()``
+to process the default values, making its behavior consistent with the
+rest of the parser.  Non-string keys and values in the defaults
+dictionary are now being implicitly converted to strings.
+(Contributed by James Tocknell. in :issue:`23835`.)
+
+The *filename* parameter of the
+:meth:`ConfigParser.read() <configparser.ConfigParser.read>` method now
+accepts bytes objects.
+(Contributed by Vincent Michel. in :issue:`31307`.)
+
+
 contextlib
 ----------
 
-:func:`~contextlib.asynccontextmanager` and
-:class:`~contextlib.AbstractAsyncContextManager` have been added. (Contributed
+The new :func:`~contextlib.nullcontext` is a simpler and faster alternative
+to :class:`~contextlib.ExitStack` for handling simple context managers.
+(Contributed by Jesse-Bakker in :issue:`10049`.)
+
+The new :func:`~contextlib.asynccontextmanager` and
+:class:`~contextlib.AbstractAsyncContextManager` have been added to
+complement their synchronous counterparts.  (Contributed
 by Jelle Zijlstra in :issue:`29679` and :issue:`30241`.)
 
-:class:`contextlib.AsyncExitStack` has been added. (Contributed by
+The new :class:`contextlib.AsyncExitStack` class has been added as an
+asynchronous equivalent to :class:`contextlib.ExitStack`.  (Contributed by
 Alexander Mohr and Ilya Kulakov in :issue:`29302`.)
 
+
 cProfile
 --------
 
 :mod:`cProfile` command line now accepts ``-m module_name`` as an alternative
 to script path.  (Contributed by Sanyam Khurana in :issue:`21862`.)
 
+
 crypt
 -----
 
-Added support for the Blowfish method.
+Added support for the Blowfish hashing method.
 (Contributed by Serhiy Storchaka in :issue:`31664`.)
 
 The :func:`~crypt.mksalt` function now allows to specify the number of rounds
 for hashing.  (Contributed by Serhiy Storchaka in :issue:`31702`.)
 
+
+csv
+---
+
+A single empty field is now always quoted when written into a CSV
+file.  This allows to distinguish an empty row from a row consisting of
+a single empty field.
+(Contributed by Licht Takeuchi. in :issue:`32255`.)
+
+
 datetime
 --------
 
-Added the :meth:`datetime.fromisoformat <datetime.datetime.fromisoformat>`
+Added the :meth:`datetime.fromisoformat() <datetime.datetime.fromisoformat>`
 method, which constructs a :class:`~datetime.datetime` object from a string
 in one of the formats output by
-:meth:`datetime.isoformat <datetime.datetime.isoformat>`.
+:meth:`datetime.isoformat() <datetime.datetime.isoformat>`.
 (Contributed by Paul Ganssle in :issue:`15873`.)
 
+The :class:`tzinfo <datetime.tzinfo>` objects now supprt sub-minute offsets.
+(Contributed by Alexander Belopolsky in :issue:`5288`.)
+
+
+dbm
+---
+
+:func:`dbm.dumb` now supports reading read-only files and no longer writes the
```

Comment:
Not in this section. This should be in two sections: a part in minor enhancements, and a part in deprecations.

``` :mod:`dbm.dumb` ``` instead of ``` :func:`dbm.dumb` ```.